.github/ISSUE_TEMPLATE/add_plugin.md

```diff
@@ -18,36 +18,30 @@ A brief description what your plugin does. Consider including screenshots to hel
 
 #### Review commit
 <!--
-Please link to an open source repository and release that should be used for review. As Mattermost code reviews and builds all plugins itself when listing in the Marketplace, the link cannot point at an already built plugin.
+Please link to an open source repository and release that should be used for review. As Mattermost code reviews and builds all plugins itself when listing in the Marketplace, the link cannot point at an already-built plugin.
 -->
 
 ## Checklist
 <!--
 Go through this checklist and confirm every item.
-
-It's fine if your plugin doesn't fulfill every item (e.g. it isn't production ready yet). You can still submit it!
-You can also still do code changes while the plugin is in review and fix issues on the fly.
-
-Even if your plugin isn't production-ready, it might still be added to the Marketplace as "Beta".
-See https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/#beta-plugins for more details.
-
-If your plugin isn't production ready, please leave a comment stating whether you plan to fulfill the whole checklist or submit a "Beta" plugin.
+It's fine if your plugin doesn't fulfill every item (e.g. it isn't production ready yet). You can still submit it! You can also still do code changes while the plugin is in review and fix issues on the fly.
```

Comment:
The line breakes were placed intentionally here they ensure the text is readable in the "Create Issue" text box without scrolling. Could you please revert the changes?